PowerVCF.psm1

```diff
@@ -2180,11 +2169,10 @@ Function Remove-VCFLicenseKey {
 }
 Export-ModuleMember -Function Remove-VCFLicenseKey
 
-######### End APIs for managing License Keys ##########
-
+#EndRegion APIs for managing License Keys
 
 
-######### Start APIs for managing Members of the Federation ##########
+#Region APIs for managing Members of the Federation
```

Comment:
Suggest merging this with #Region APIs for managing Federation. We should also consider deprecating these functions

Reply:
I can regroup the functions no problem. 

How have you dealt with deprecating functions in the past ? put a banner to the user?

Reply:
I just checked and we already have a deprecation notice in the functions so i think we are good. Leaving the functions in place will avoid breaking backwards compatibility

"Write-Warning "Multi-Instance Management has been deprecated in VMware Cloud Foundation v4.4.0 and later, this API is no longer supported"